src/rules/completedDocsRule.ts

```diff
@@ -38,6 +38,7 @@ export const ARGUMENT_VARIABLES = "variables";
 
 export const DESCRIPTOR_TAGS = "tags";
 export const DESCRIPTOR_LOCATIONS = "locations";
+export const DESCRIPTOR_OVERLOADS_SEPARATE_DOCS = "overloads-separate-docs";
```

Comment:
can we just name it "overloads"? I think seeing that in the config in context will be clear enough